mlir/lib/Driver/CompilerDriver.cpp

```diff
@@ -0,0 +1,459 @@
+// Copyright 2023 Xanadu Quantum Technologies Inc.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+
+//     http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <filesystem>
+#include <list>
+#include <memory>
+#include <string>
+#include <unordered_map>
+
+#include "gml_st/transforms/passes.h"
```

Comment:
Ah I was curious why this showed up in other places, but I guess we register it just in case but don't make use of it at the moment.

Reply:
We do use it! We use if for `scalarize`.

Reply:
Ah, but maybe we don't need `include`. Got it.

Reply:
Ah got it, didn't realize it that pass came from the `gml_st` dialect 👍